2 dz/6/bullsAndCows.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+#include "animalCounter.h"
+#include <iostream>
```

Comment:
Are you sure you need to include these implementation details in the header file? I mean libraries. You can change <iostream> to <stdio.h>, for example, and your interface here won't change.